src/components/structures/LoggedInView.js

```diff
@@ -399,10 +399,11 @@ const LoggedInView = createReactClass({
             ev.preventDefault();
         } else if (!hasModifier) {
             const isClickShortcut = ev.target !== document.body &&
-                (ev.key === "Space" || ev.key === "Enter");
```

Comment:
Also I guess this line was buggy earlier, as `ev.key` is `" "` but `ev.code` is `"Space"` so it never worked